unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Main.cs

```diff
@@ -18,6 +18,8 @@ public class Main : MonoBehaviour
 
         public DebugConfig debugConfig;
 
+        private PerformanceMetricsController performanceMetricsController;
```

Comment:
`PerformanceMetricsController` is an implementation detail and nobody will reference it, so moved this away from `Environment` onto `Main`.